config/config_bsd.go

```diff
@@ -7,11 +9,10 @@ import (
 
 var mackerelRoot = filepath.Join(os.Getenv("HOME"), "Library", getAgentName())
```

Comment:
Current {free,net}bsd also uses this directory as root directory, but I think this is not good for {free,net}bsd.
(I guess current `mackerelRoot` of {free,net}bsd is just copied from macOS)
Isn't it better to set somewhere more *nix-friendly one?